### PR TITLE
Removed /api prefix from endpoints, and add /users prefix to auth

### PR DESCRIPTION
**Pull request by dlarocque**
`/api` prefix is redundant, since the URL of the frontend and backend are different. 
Added `/users` prefix to auth API

Some minor style changes in the test plan, too (I guess MD formatter didn't fix them before)

### CODE REVIEW

File: documentation/test-plan.md [hunk 1]
```diff
@@ -48,73 +48,73 @@ This test plan covers unit, integration, acceptance, regression, and load testin
...
+   1. Return `201 Created` upon POST request to `/auth/register` with valid payload.
+   2. Return `400 Bad Request` upon POST request to `/auth/register` with an already used username.
+   3. Return `200 OK` and a JWT upon POST to `/auth/login` with correct credentials.
+   4. Return `400 Bad Request` upon POST to `/auth/login` with incorrect credentials.
+   5. Return `200 OK` upon POST to `/auth/reset-password` with the correct security answer.
+   6. Return `400 Bad Request` upon POST to `/auth/reset-password` with the incorrect security answer.
+   7. Return `400 Bad Request` upon POST to `/auth/reset-password` with a missing payload.
+   8. Return `200 OK` upon PUT to `/account` with valid new user data.
+   9. Return `400 Bad Request` upon PUT to `/account` with a new username that already exists.
+   10. Return `200 OK` upon DELETE to `/account` from authenticated session.
```
Comment (BarbzCodez) [inline documentation/test-plan.md:60]: shouldn't these be `/users` be in these as well or am I dumb (?)

Comment (dlarocque) [reply documentation/test-plan.md:60]: omg ty yes you're right, will fix

**Review by BarbzCodez — APPROVED** [review]
looks good :)

**Review by VictoriaKGN — APPROVED** [review]
image

**Pull request merged**